<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareFile</class>
 <widget class="QWidget" name="CompareFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompareFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="file_text1"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_open_1">
       <property name="text">
        <string>打开文件1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="file_text2"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_open_2">
       <property name="text">
        <string>打开文件2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="listWidget1"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_compare">
     <property name="text">
      <string>比较文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
